revelio/urls.py

```diff
@@ -44,6 +44,7 @@
     path('vendors/', include('vendor.urls')),
     path('tasks/', include('task.urls')),
     path('task-steps/', include('task_steps.urls')),
+    path('rundown/', include('rundown.urls')),
```

Comment:
to be more consistent, please add s at the end of rundown

Reply:
Noted